manager/integration/tests/test_rwx.py

```diff
@@ -0,0 +1,152 @@
+
+    3. Wait for longhorn-nfs-provisioner deployment to come up healthy.
+    4. Verify the volume named as the PV attached to longhorn-nfs-provisioner
+       `volume.ready == True`
+    5. Create a StatefulSet of 1 pod with VolumeClaimTemplate and
+       longhorn-nfs class in default namespace.
+    6. Wait for StatefulSet to come up healthy.
+    7. Write data and compute md5sum.
+    8. Create another statefulset with same pvc which got created with first
+       statefulset.
+    9. Wait for statefulset to come up healthy.
+    10. Check the data md5sum.
+    11. Write more data and compute md5sum
+    12. Check the data md5sum in longhorn-nfs-provisioner volume
+    """
+    pass
+
+
+@pytest.mark.skip(reason="TODO")
+def test_rwx_parallel_writing():
+    """
+    Test parallel writing of data
+
+    1. Create PodSecurityPolicy for longhorn-nfs-provisioner as sample yaml
+       https://raw.githubusercontent.com/longhorn/longhorn/master/examples
+       /rwx/01-security.yaml
+
+    2. Create serviceAccount, service, longhorn-nfs-provisioner deployment,
+       PVC and StorageClass as sample yaml https://raw.githubusercontent.com/
+       longhorn/longhorn/master/examples/rwx/02-longhorn-nfs-provisioner.yaml
+
+    3. Wait for longhorn-nfs-provisioner deployment to come up healthy.
+    4. Verify the volume named as the PV attached to longhorn-nfs-provisioner
+       `volume.ready == True`
+    5. Create a StatefulSet of 1 pod with VolumeClaimTemplate and
+       longhorn-nfs class in default namespace.
+    6. Wait for StatefulSet to come up healthy.
+    7. Create another statefulset with same pvc which got created with first
+       statefulset.
+    8. Wait for statefulset to come up healthy.
+    9. Start writing 800 MB data in first stateful set and start writing 500 MB
+       data in second stateful set.
+    10. Compute md5sum.
+    11. Check the data md5sum in longhorn-nfs-provisioner volume
+    """
+    pass
+
+
+@pytest.mark.skip(reason="TODO")
+def test_rwx_statefulset_scale_down_up():
+    """
+    Test Scaling up and down of pods attached to longhorn-nfs-provisioner
+    volume.
+
+    1. Create PodSecurityPolicy for longhorn-nfs-provisioner as sample yaml
+       https://raw.githubusercontent.com/longhorn/longhorn/master/examples
+       /rwx/01-security.yaml
+
+    2. Create serviceAccount, service, longhorn-nfs-provisioner deployment,
+       PVC and StorageClass as sample yaml https://raw.githubusercontent.com/
+       longhorn/longhorn/master/examples/rwx/02-longhorn-nfs-provisioner.yaml
+
+    3. Wait for longhorn-nfs-provisioner deployment to come up healthy.
+    4. Verify the volume named as the PV attached to longhorn-nfs-provisioner
+       `volume.ready == True`
+    5. Create a StatefulSet of 2 pods with VolumeClaimTemplate and
+       longhorn-nfs class in default namespace.
+    6. Wait for StatefulSet pods to come up healthy.
+    7. Write data and compute md5sum in both pods
+    8. Force delete pods.
```

Comment:
What does force delete refer to in this case?

Reply:
Actually force delete is not required, a delete will do. I wrote it to make sure this delete pod in any case.